src/nimib.nim

```diff
@@ -64,13 +66,30 @@ template nbInit*() =
     nbTextBlock(nbBlock, nbDoc, body)
 
   template nbCode(body: untyped) =
-    nbCodeBlock(nbBlock, nbDoc, body)
+    nbCodeBlock(nbSource, nbBlock, nbDoc, body)
 
-  template nbCodeInBlock(body: untyped) =
+  template nbCodeInBlock*(body: untyped): untyped =
     block:
       nbCode:
         body
 
+#[
```

Comment:
this (and other similar commented code) can probably be removed when everything is tested, correct?